code/game/objects/items/weapons/storage/bible.dm

```diff
@@ -68,6 +68,38 @@
 
 	var/datum/role/vampire/V = isvampire(user)
 
+	//they have holy water in them? deconversion mode activate! anyone can do it.
+	if (M.reagents?.has_reagent(HOLYWATER) && user.a_intent == I_HELP)
+		if (M.stat == DEAD)
+			to_chat(user,"<span class='warning'>You cannot deconvert the dead!</span>")
+			return 1
+		if (M.health < 20)
+			to_chat(user,"<span class='warning'>\The [M] is too weak to handle the deconversion ritual, patch them up a bit first.</span>")
```

Comment:
Put the "you need to have this much HP before deconvering" into the changelog, sneaky c*der.

Reply:
I figured people would figure it out themselves and the changelog is bloated enough already but you're the boss.

Reply:
Considering the specific health threshold is hidden in the code, you might as well make it detailed. Also what's more bloat too a bloated changelog, anyway?

Reply:
I think there's a more valid conversation here that the changelog is so long that most peoples' eyes will glaze over. I would go as far to say we need a tl;dnr version with the absolute highlights.